src/operations/measures/measureBounds.js

```diff
@@ -0,0 +1,107 @@
+const flatten = require('../../utils/flatten')
+
+const {vec2, vec3} = require('../../math')
+
+const {geom2, geom3, path2, poly3} = require('../../geometry')
+
+/*
+ * Measure the min and max bounds of the given (path2) geometry.
+ * @return {[[x,y,z], [x,y,z]]} the bounds (min and max points) for the geometry
+ */
+const measureBoundsOfPath2 = (geometry) => {
+  const points = path2.toPoints(geometry)
+
+  let minpoint
+  if (points.length === 0) {
+    minpoint = vec2.create()
+  } else {
+    minpoint = vec2.clone(points[0])
+  }
+  let maxpoint = vec2.clone(minpoint)
+
+  points.forEach((point) => {
+    vec2.min(minpoint, minpoint, point)
+    vec2.max(maxpoint, maxpoint, point)
+  })
+  minpoint = [minpoint[0], minpoint[1], 0]
+  maxpoint = [maxpoint[0], maxpoint[1], 0]
+
+  return [minpoint, maxpoint]
+}
+
+/*
+ * Measure the min and max bounds of the given (geom2) geometry.
+ * @return {[[x,y,z], [x,y,z]]} the bounds (min and max) for the geometry
+ */
+const measureBoundsOfGeom2 = (geometry) => {
+  const points = geom2.toPoints(geometry)
+
+  let minpoint
+  if (points.length === 0) {
+    minpoint = vec2.create()
+  } else {
+    minpoint = vec2.clone(points[0])
+  }
+  let maxpoint = vec2.clone(minpoint)
+
+  points.forEach((point) => {
+    vec2.min(minpoint, minpoint, point)
+    vec2.max(maxpoint, maxpoint, point)
+  })
+  minpoint = [minpoint[0], minpoint[1], 0]
+  maxpoint = [maxpoint[0], maxpoint[1], 0]
+
+  return [minpoint, maxpoint]
+}
+
+/**
+ * Measure the min and max bounds of the given (geom3) geometry.
+ * @return {[[x,y,z], [x,y,z]]} the bounds (min and max) for the geometry
+ */
+const measureBoundsOfGeom3 = (geometry) => {
```

Comment:
I still have doubts about puting operations across different types in the same modules, but let's keep for now